### PR TITLE
Renaming to remove the words Master and Slave from the codebase

### PR DESCRIPTION
**Pull request by alanfx**
All references to MASTER, Master, and master have been changed to MAIN,
Main, and main.
All references to SLAVE, Slave, and slave have been changed to WORKER,
Worker, and worker.

There are still references to master and slave related to the
terminology in Jenkins

RadarGun PR: https://github.com/radargun/radargun/pull/670

### CODE REVIEW

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -21,17 +21,17 @@
 == Release 1.2 (2016-10-19)
 
 * https://github.com/vjuranek/radargun-plugin/issues/58[#58] Added Script source deprecation warning
-* https://github.com/vjuranek/radargun-plugin/issues/52[#52] Removed deprecation from JVM option on master node
```
Comment (vjuranek) [inline CHANGELOG.adoc:24]: not sure if it worth to change it in the changelog, but no problem with it

Comment (alanfx) [reply CHANGELOG.adoc:24]: Yeah, let me take these changes out

**Comment by alanfx** [comment]
> LGTM, changes in RG itself are not merge yet. Can I expect they will be merged soon? (I mean: can I merge the PR?)

Yes, there needs to be a bit of coordination here! Let me push again without the changes to the CHANGELOG, and then you can merge this one. Once this one is merged and a new plugin is released, I can get the RadarGun changes merged and possibly finally release 3.0.0! Thanks, Vojta

**Comment by alanfx** [comment]
OK, updated

**Comment by vjuranek** [comment]
Merged (with some additional changes), thanks!

**Comment by vjuranek** [comment]
release as 1.9

**Comment by alanfx** [comment]
Thanks, Vojta!

**Pull request closed without merging**